build.xml

```diff
@@ -271,17 +271,6 @@
         </exec>
     </target>
 
-    <target name="domains-urls-check" description="Checks the domain urls configuration.">
```

Comment:
I am thinking that it would be probably nice to link the original pull requests on the 2nd line of your commit message :slightly_smiling_face: 

"- see https://github.com/shopsys/shopsys/pull/540"